modules/3d-tiles/src/parsers/parse-3d-tile-point-cloud.js

```diff
@@ -50,6 +48,77 @@ function extractPointCloud(tile) {
   tile.isRGB565 = false;
   tile.isOctEncoded16P = false;
```

Comment:
Maybe `Object.seal(this)` here to make sure we have initialized all values with defaults.